src/components/RiddleCard.tsx

```diff
@@ -22,14 +21,7 @@ import type { FC } from 'react';
 import type { CountryCode } from '../utils/CountryCodes';
 import type { Difficulty } from '../utils/Difficulty';
```

Comment:
`CountryCode` is declared but its value is never read.
`Difficulty` is declared but its value is never read.